src/context/datacontext.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const DataContext = React.createContext();
+
+export default DataContext;
```

Comment:
Where is this being used?

Reply:
Oh, I forgot I had done this before regarding the search component on the page. I will remove it with my next commit